Cython/Compiler/Optimize.py

```diff
@@ -3463,6 +3463,20 @@ def _handle_simple_method_object___truediv__(self, node, function, args, is_unbo
     def _handle_simple_method_object___div__(self, node, function, args, is_unbound_method):
         return self._optimise_num_div('Divide', node, function, args, is_unbound_method)
 
+    _handle_simple_method_int___add__ = _handle_simple_method_object___add__
+    _handle_simple_method_int___sub__ = _handle_simple_method_object___sub__
+    _handle_simple_method_int___mul__ = _handle_simple_method_object___mul__
+    _handle_simple_method_int___eq__ = _handle_simple_method_object___eq__
+    _handle_simple_method_int___ne__ = _handle_simple_method_object___ne__
+    _handle_simple_method_int___and__ = _handle_simple_method_object___and__
+    _handle_simple_method_int___or__ = _handle_simple_method_object___or__
+    _handle_simple_method_int___xor__ = _handle_simple_method_object___xor__
+    _handle_simple_method_int___rshift__ = _handle_simple_method_object___rshift__
+    _handle_simple_method_int___lshift__ = _handle_simple_method_object___lshift__
+    _handle_simple_method_int___mod__ = _handle_simple_method_object___mod__
+    _handle_simple_method_int___floordiv__ = _handle_simple_method_object___floordiv__
+    _handle_simple_method_int___truediv__ = _handle_simple_method_object___truediv__
+
```

Comment:
Just curious about why you removed `__div__` in the second commit? (Not suggesting you should spend time making it work if it doesn't though... just asking what doesn't work)

Reply:
I added it first but then removed because:
1. it does not optimisation, if I am not wrong
2. it is only for python2 so I decided to simplify the PR

Reply:
Please let me know if you think it should be present though I would added but for me it is not worth it.

Reply:
No that seems fine. I think we can skip this for Python 2. I hadn't realised they'd changed the names in Python 3.